.github/workflows/pre-commit.yml

```diff
@@ -27,6 +27,15 @@ jobs:
         run: pip install pre-commit
       - name: Run pre-commit
         run: pre-commit run --all-files --show-diff-on-failure --color=always
+        env:
+          # Consider valid a PR that changes README fragments but doesn't
+          # change the README.rst file itself. It's not really a problem
+          # because the bot will update it anyway after merge. This way, we
+          # lower the barrier for functional contributors that want to fix the
+          # readme fragments, while still letting developers get README
+          # auto-generated (which also helps functionals when using runboat).
+          # DOCS https://pre-commit.com/#temporarily-disabling-hooks
+          SKIP: oca-gen-addon-readme
```

Comment:
@AungKoKoLin1997 Can you please explain the implication of this.

Reply:
This is to skip generating READEME.rst in pre-commit CI. Actually, this is more related with OCA process. In our case, we will run pre-commit in our env before we push to git.
Related - https://github.com/OCA/oca-addons-repo-template/pull/212

<img width="1059" alt="Screen Shot 2023-10-16 at 08 53 40" src="https://github.com/qrtl/hls-custom/assets/45355704/bf9e6c3d-30c9-499b-b6d7-b3c04fd783ee">